Mathlib/Algebra/Group/Pi/Basic.lean

```diff
@@ -548,6 +548,18 @@ theorem bijective_pi_map {F : ∀ i, f i → g i} (hF : ∀ i, Bijective (F i))
   ⟨injective_pi_map fun i => (hF i).injective, surjective_pi_map fun i => (hF i).surjective⟩
 #align function.bijective_pi_map Function.bijective_pi_map
 
+lemma comp_eq_const_iff {α β γ : Type*} (b : β) (f : α → β) {g : β → γ}
```

Comment:
I haven't tested this, but I wonder if these types are already defined at the top of the file.
```suggestion
lemma comp_eq_const_iff (b : β) (f : α → β) {g : β → γ}
```

Reply:
Yes, they are